api/controllers/v1/document/find-by-caver-id.js

```diff
@@ -17,7 +17,7 @@ module.exports = async (req, res) => {
   const whereClause = { author: caverId, isDeleted: false };
 
   const totalDocuments = await TDocument.count().where(whereClause);
-  const documents = await DocumentService.appendPopulateForSimpleDocument(
```

Comment:
Do we really want to have the full document here? 
Has it can be a lot of objects for a single caver it may be better only to retrieve simple documents (and use `toSimpleDocument` as a converter)

Also, as this route is no longer used in the front, maybe it can be removed entirely?